public/app/features/plugins/PluginPage.tsx

```diff
@@ -293,13 +298,48 @@ class PluginPage extends PureComponent<Props, State> {
     );
   }
 
+  renderPluginNotice() {
+    const { plugin } = this.state;
+
+    if (!plugin) {
+      return null;
+    }
+
+    if (plugin.meta.signature === PluginSignatureStatus.internal) {
+      return null;
+    }
+
+    return (
+      <InfoBox
+        aria-label={selectors.pages.PluginPage.signatureInfo}
+        severity={plugin.meta.signature !== PluginSignatureStatus.valid ? 'warning' : 'info'}
+        urlTitle="Read more about plugins signing"
+        url="TODO"
```

Comment:
@daniellee do we have a doc about that already?

Reply:
@marcusolsson ^

Reply:
https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/